pkg/list/lister.go

```diff
@@ -101,27 +178,47 @@ func (l *lister) fetchRepoAndInstalled(ctx context.Context, options ListOptions)
 
 	wg.Wait()
 
-	compositeErr := multierr.Combine(repoErr, pkgErr, pkgInfoErr)
-	if pkgErr != nil || pkgInfoErr != nil {
+	compositeErr := multierr.Combine(repoErr, clPkgErr, pkgErr, pkgInfoErr)
+	if clPkgErr != nil || pkgErr != nil || pkgInfoErr != nil {
```

Comment:
Already was like this, but I just realized that `repoErr` is not considered here. Should it be?

Reply:
I believe that was on purpose, because it could be that one of the repositories is not available, but the rest could still be displayed – or am I mixing this up with something else?

Reply:
Yes I think you're right. Maybe there should be a comment explaining this :sweat_smile: